subsys/bluetooth/host/l2cap_br.c

```diff
@@ -577,22 +579,23 @@ static enum l2cap_br_conn_security_result
 l2cap_br_conn_security(struct bt_l2cap_chan *chan, const uint16_t psm)
 {
 	int check;
+	struct bt_l2cap_br_chan *ch = BR_CHAN(chan);
```

Comment:
Ditto here, I think that `br_chan` may be a better name

Reply:
The necessity is not great